src/main/java/com/pinterest/secor/message/Message.java

```diff
@@ -20,55 +20,75 @@
 import java.io.OutputStream;
 import java.lang.String;
 
+
 /**
  * Message represents a raw Kafka log message.
  *
  * @author Pawel Garbacki (pawel@pinterest.com)
  */
 public class Message {
+
```

Comment:
You can make local changes and push to the review branch which I believe is estezz:master